deltachat-ios/AppDelegate.swift

```diff
@@ -445,24 +450,47 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
     // MARK: - misc.
 
-    func openDatabase() {
-        guard let databaseLocation = DatabaseHelper(dcContext: dcContext).updateDatabaseLocation() else {
+    func migrateToDcAccounts() {
+        let dcContext = DcContext()
+
+        // first database migration to shared container, can be removed at some point
+        // implemented in April 2020 (https://github.com/deltachat/deltachat-ios/pull/612)
+        let databaseHelper = DatabaseHelper(dcLogger: dcContext.logger)
+        guard let databaseLocation = databaseHelper.updateDatabaseLocation() else {
             fatalError("Database could not be opened")
         }
-        logger.info("open: \(databaseLocation)")
-        dcContext.openDatabase(dbFile: databaseLocation)
+
+        if databaseHelper.unmanagedDatabaseLocation == nil {
+            return
+        }
+
+        if dcAccounts.migrate(dbLocation: databaseLocation) == 0 {
+            fatalError("Account could not be migrated")
+            // TODO: show error message in UI
+        }
+        databaseHelper.clearUnmanagedAccountData()
```

Comment:
`dcAccounts.migrate()` has cleared/moved the unmanaged account data, this function should not be needed.

Reply:
cool, good to know :)